packages/array-mode/src/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Computes the mode of the given values
+ * @param {Array<number>} input
+ * @return {number}
+ */
+export default function mode(input) {
```

Comment:
I did this jsperf (https://jsperf.com/mode) if you can get better results just tell me, (using sort was faster before adding the copy)